src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.module.ts

```diff
@@ -7,15 +7,6 @@ import { UserRouteAccessService } from 'app/core/auth/user-route-access-service'
 import { ArtemisPlagiarismCasesSharedModule } from 'app/course/plagiarism-cases/shared/plagiarism-cases-shared.module';
 
 const routes: Routes = [
-    {
```

Comment:
You moved this route into another module. Why? Was this intentional?

Reply:
This was intentional. Otherwise, I would have had to add the CourseManagementTabBarComponent here again. Since the route is a direct child of course-management, the change seems reasonable to me.

Also, I believe I had a problem with nested empty parts.